### ISSUE TITLE
Actions editor: ability to set value to service independently from control

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3072]
Currently, you can set a service value only from a control.

I would like to be able to use a constant or an XPath expression as well.

**Comment by ebruchez** [comment]
[+1 from customer](https://3.basecamp.com/3600924/buckets/2567696/messages/756536836)

**Comment by ebruchez** [comment]
In "Service Request Actions", have:

- "Control" and current control selection dropdown
- "Formula" and text field for formula

**Comment by ebruchez** [comment]
The format of actions should also be changed. We have:

```xml
<xf:action class="fr-set-service-value-action">
    <xf:var name="control-name" value="'my-control'"/>
    <xf:var name="path" value="/my/path"/>
</xf:action>
```

And we need to allow:

```
<xf:action class="fr-set-service-value-action">
    <xf:var name="expression" value="my-xpath-expression"/>
    <xf:var name="path" value="/my/path"/>
</xf:action>
```

**Comment by avernet** [comment]
[+1 from user](https://3.basecamp.com/3600924/buckets/7545500/messages/1250422281)

**Comment by ebruchez** [comment]
Turns out we entered the duplicate #4955 and implemented it for 2021.1. Closing this one.
